### PR TITLE
Fix 2178

### PR DESCRIPTION
**Pull request by srazi**

### CODE REVIEW

**Comment by iondissonance** [comment]
Thanky you very much!

File: src/lib/other/checkboxdialog.cpp [hunk 1]
```diff
@@ -16,47 +17,28 @@
...
-    ui->iconLabel->setPixmap(icon.pixmap(48, 48));
-    ui->iconLabel->setFixedWidth(48);
-}
-
-void CheckBoxDialog::setText(const QString &text)
-{
-    ui->textLabel->setText(text);
+    setCheckBox(m_checkBox);
+    setStandardButtons(buttons);
+    setWindowFlags(windowFlags() | Qt::MSWindowsFixedSizeDialogHint | Qt::WindowTitleHint | Qt::WindowSystemMenuHint | Qt::WindowCloseButtonHint);
```
Comment (nowrep) [inline src/lib/other/checkboxdialog.cpp]: Can you make this only for Q_OS_WIN? I think it may not work that well on other platforms (mainly on Wayland).

Comment (srazi) [inline src/lib/other/checkboxdialog.cpp]: Did you mean inheriting from QMessageBox or `setWindowFlags()`? As `setWindowFlags()` uses what was used in old `CheckBoxDialog` code.

Comment (nowrep) [inline src/lib/other/checkboxdialog.cpp]: Ah, true, sorry.

**Pull request merged**